release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+
+Previous values are:
+
+    template_service_broker_prefix="registry.example.com/"
+    template_service_broker_image_name="origin-template-service-broker" or template_service_broker_image_name="ose-template-service-broker"
+
+New values are:
+    template_service_broker_prefix="registry.example.com/ose-" or template_service_broker_prefix="registry.example.com/origin-"
+    template_service_broker_image_name="template-service-broker"
+
+
+[discrete]
+[[removed-become-no-instances]]
+=== Removed several instances of 'become: no' on certain tasks and playbooks inside of openshift-anisble
+In an effort to provide greater flexibility for users, we have removed several instances of 'become: no' on certain tasks and playbooks inside of openshift-anisble. These statements were primarily applied on 'local_action' and 'delegate_to: localhost' commands for creating temporary files on the host running ansible.
+
+If a user is running ansible from a host that does not allow passwordless sudo, some of these commands may fail of the user runs ansible-playbook with the '-b' (become) command line switch, or has 'ansible_become=True' applied to localhost in inventory or group_vars.
+
+Elevated permissions are not requried on the localhost when running openshift-ansible plays.
+
+If target hosts (where openshift is being deployed) require the use of become, it is recommended to add "ansible_become=True" for those hosts or groups in inventory or group_vars/host_vars.
+
+If the user is running as root on the localhost, or connection to the root user on the remote hosts instead of using become, no change should be noticed.
+
+[discrete]
+[[unqualified-image-specs]]
+===
+unqualified image specs now default to docker.io and require apiserver configuration to resolve to different registries.
+
+WIP in https://bugzilla.redhat.com/show_bug.cgi?id=1518378 to determine what this config looks like and how we will migrate existing unqualified pods
+
+[discrete]
+[[SchedueldJob-objects-not-supported]]
+=== batch/v2alpha1 SchedueldJob objects are no longer supported
+
+batch/v2alpha1 SchedueldJob objects are no longer supported. Use CronJobs instead.
+
+[discrete]
+[[autoscaling-API-group-removed]]
+===  The autoscaling/v2alpha1 API group has been removed
+The autoscaling/v2alpha1 API group has been removed
+
+[discrete]
+[[start-node-requires-swap-to-be-disabled]]
+The openshift start node requires swap to be disabled. This is already done as part of the ansible node install.
+
+
+[[ocp-39-bug-fixes]]
+== Bug Fixes
+
+This release fixes bugs for the following components:
+
+*Authentication*
+
+*
+
+*Builds*
+
+
+*Command Line Interface*
+
+
+*Containers*
+
+
+*Deployments*
+
+
+*Images*
+
+*Image Registry*
+
+
+*Installer*
+
+*Logging*
+
+
+*Web Console*
+
+
+*Master*
+
+
+*Metrics*
+
+
+*Networking*
+
+*Pod*
+
+
+*Routing*
+
+
+*Service Broker*
+
+
+*Storage*
+
+
+*Templates*
+
+
+*Upgrade*
```

Comment:
All of these sections are blank, do you need a BZ list to include for these sections?